listenbrainz/db/user_setting.py

```diff
@@ -0,0 +1,101 @@
+import sqlalchemy
+from datetime import datetime, timedelta
+from listenbrainz import db
+from listenbrainz.db.exceptions import DatabaseException
+
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Do not create loggers here. Instead use current_app.logger everywhere. You can look into other db files for reference.